duckduckgo/iOS
# Sync paused alert

Task/Issue URL: https://app.asana.com/0/0/1205842709739243/f
Tech Design URL: https://app.asana.com/0/1204186595873227/1205715582565106/f

**Description**: Alert user the first time bookmark sync and credentials sync is paused due to too many items or large request.



**Steps to test this PR**:
1. Create a sync account
2. Use proxy to return 409 and 413 for bookmarks
3. An alert should be displayed warning the user (clicking on ok will dismiss the alert clicking on learn more should lead to the sync settings)
4. The alert should then not be visible until the problem is solved.
5. Check the sync settings menu item shows the danger sign.
6. If sync is successful the danger symbol should disappear
7. If the error is returned again the alert should be displayed
8. Execute the same steps for credentials
9. The alert Should NOT be seen when the error occurs when activating sync for the first time.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
